python/tchannel/rw.py

```diff
@@ -347,7 +347,7 @@ class NumberReadWriter(ReadWriter):
         1: '>B',
         2: '>H',
         4: '>I',
-        8: '>Q',
+        8: '>q',
```

Comment:
in the protocol, it says clearly it is i64. That's why I change it. It will only affect the id which are using 8 bytes.